src/components/avatar.tsx

```diff
@@ -1,18 +1,23 @@
 import React from 'react';
-import { FiTwitter, FiLinkedin } from "react-icons/fi";
+import { FiTwitter, FiLinkedin, FiGithub } from "react-icons/fi";
```

Comment:
Everything in this file is good, just wondering why this branch changes `avatar.tsx`  (my PRs aren't any cleaner).  :p